lola.html

```diff
@@ -507,17 +507,21 @@ <h3>Feature Discovery</h3>
     "accountPortabilityOauth": "https://example.com/oauth2/porting-access-endpoint",
     "inbox": "https://oakfrost.example.com/brock/inbox",
     "outbox": "https://oakfrost.example.com/brock/outbox",
-    "content": "https://oakfrost.example.com/brock/content",
     "following": "https://oakfrost.example.com/brock/following",
     "followers": "https://oakfrost.example.com/brock/followers",
     "liked": "https://oakfrost.example.com/brock/liked",
     "blocked": "https://oakfrost.example.com/brock/blocked",
-    "migration": "https://oakfrost.example.com/brock/outbox"
+    "migration": {
```

Comment:
Since "following" and "blocked" are important for migration, should these URLs also be copied (or moved) into the "migration" collection?  From what I've seen, servers aren't already putting those collections on the Actor object - the Actor object is normally accessed by other servers in the fediverse, and the way the user adds following and blocked accounts is for example using Mastodon's client API instead of ActivityPub.  But I made sure these are required for LOLA.

Reply:
Different servers do different things, so it would have to depend on which specific app we're talking about :(

You could make a case for duplicating any available collection into the actor's "migration" property, which would signal affirmatively that "this collection is safe to migrate".  If the server intends to just use the regular collection, then it would be really easy to just duplicate the original ID under "migration" too.  Plus, this would give target servers a single place to look for all of their data.

I'm super new here, so I don't want to rock the boat without checking in before I make any big changes. But if this looks good to you, I can certainly rewrite this PR so that "following" and "blocked" are also supposed to be sourced from the actor's "migration" property.

Reply:
(sorry it took me a bit to respond here)

Yes. Fewer changes to the design doc are better. And there's no rush to merge my changes until everyone is happy with them.

If `Following` and `Blocked` are available for migration, then I think I'll recommend putting them in this property as well.  Like you said, servers are already inconsistent about what they display in Actor's profiles, so I think it would be good to make the `migration` section as clear and well-defined as possible.

And if a server has special requirements, they can always define a custom schema and put that in the `migration` property as well.

Reply:
Yes I agree with the argument that copying collections that the source things are important for migrations into the migrations property is useful. It's more clear than just having them in one or the other place.  Why don' t you merge with or without that change, and we can copy block/follow collection URLs in there too.

Reply:
@lisad I've just (finally) posted this change.  Thanks for being patient with me :)